equinox/serialisation.py

```diff
@@ -9,8 +9,43 @@
 from .custom_types import PyTree
 
 
+def _ordered_tree_map(
+    f: Callable[..., Any],
+    tree: Any,
+    *rest: Any,
+    is_leaf: Optional[Callable[[Any], bool]] = None,
+) -> Any:
+    """Like jax.tree_util.tree_map, but guaranteed to iterate over the tree
+    in fixed order. (Namely depth-first left-to-right.)
```

Comment:
I'd suggest removing all of the docstring below these two lines. It's totally fine to have one docstring simply point at another.